client/app/pages/queries/query.html

```diff
@@ -65,6 +65,7 @@ <h3>
             <li ng-if="!query.is_draft && query.id != undefined && (isQueryOwner || currentUser.hasPermission('admin'))"><a ng-click="togglePublished()">Unpublish</a></li>
             <li class="divider" ng-if="!query.is_archived"></li>
             <li ng-if="query.id != undefined"><a ng-click="showApiKey()">Show API Key</a></li>
+            <li ng-if="query.id != undefined"><a href="{{ iodideUrl }}">Open in Iodide Notebook (experimental)</a></li>
```

Comment:
Let's use a shorter list title, e.g. `Open in Iodide`. Also there's no need to mark it as experimental IMO, maybe instead add a banner to iodide.io denoting the experimental status?